spec/models/user_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  before :each do
+    @user = User.new(name: 'Test', email: 'test@test.com', password: 123_456, password_confirmation: 123_456)
+    @user.save
+  end
+
+  it 'is valid' do
+    expect(@user).to be_valid
+  end
+
+  it 'name should be present' do
+    @user.name = nil
+    expect(@user).to_not be_valid
+  end
+end
```

Comment:
- [ ] Kindly make sure that all tests written are passing, currently, they are falling on my computer, I ran the instructions in your readme and did other configurations but it is still failing, see the screenshot below ⬇️ 

![Screenshot_559](https://user-images.githubusercontent.com/69638013/206835335-1b6e3e36-b018-4b1d-a209-4f44b7f09da9.png)

Reply:
Already fixed, I had to update the rails_helper and the gemfile.lock to make it work. Thank you for everything.